pkg/systemd/overview.jsx

```diff
@@ -40,46 +39,16 @@ import { UsageCard } from './overview-cards/usageCard.jsx';
 import { SuperuserAlert } from './superuser-alert.jsx';
 import { SuperuserIndicator } from "../shell/superuser.jsx";
 import { ShutdownModal } from 'cockpit-components-shutdown.jsx';
-import { LoginMessage } from "./loginMessage.jsx";
 
 const _ = cockpit.gettext;
 
-class LoginAlert extends React.Component {
```

Comment:
I was rather confused when I read the first commit and saw it introduce this, only to see that the second commit removes it again. let's squash these two on landling.

Reply:
I kept it as two comments and want to squash it later, as I wasn't 100% sure everyone agrees on the new behaviour.

Reply:
Resolved.